src/main/java/com/exasol/releasedroid/github/GithubAPIAdapter.java

```diff
@@ -155,7 +160,50 @@ private void sendGitHubRequest(final URI uri, final String json) throws GitHubEx
     private void validateResponse(final HttpResponse<String> response) throws GitHubException {
         if ((response.statusCode() < HttpURLConnection.HTTP_OK)
                 || (response.statusCode() >= HttpURLConnection.HTTP_MULT_CHOICE)) {
-            throw new GitHubException("F-RR-GH-6: An HTTP request failed. " + response.body());
+            throw new GitHubException("F-RR-GH-6: An executing workflow HTTP request failed. " + response.body());
+        }
+    }
+
+    private String getWorkflowConclusion(final String repositoryFullName, final String workflowName)
+            throws GitHubException {
+        while (true) {
+            waitOneMinute();
```

Comment:
Create a log message before starting this that tells users that the software is now going to wait for the results and that this can take a few minutes to a couple of hours depending on the build.

Also increase the waiting period to five minutes after 10 minutes in order to keep the number of requests low. We don't want to run into trouble with GH.